Signal/src/ViewControllers/ConversationView/Cells/ConversationViewCell.h

```diff
@@ -69,12 +70,11 @@ NS_ASSUME_NONNULL_BEGIN
 // * Users enters another view (e.g. conversation settings view, call screen, etc.).
 @property (nonatomic) BOOL isCellVisible;
 
-// The width of the collection view.
-@property (nonatomic) int contentWidth;
+@property (nonatomic, nullable) ConversationLayoutInfo *layoutInfo;
```

Comment:
This PR introduces `ConversationLayoutInfo`, which gathers layout-related metadata like sizes, spacing, margins.  I suspect it will get elaborated.